### PR TITLE
50 implement scraping logic

### PR DESCRIPTION
**Pull request by Koufan-De-King**
A script to scrape data from websites was written into a new folder called 'app', into which the controller, 'app.py' was also moved.

### CODE REVIEW

File: app/scraping.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    except requests.RequestException as e:
+        print("An error occurred:")
+        print(e)
+
+    except Exception as e:
+        print(f"Error scraping website: {url}")
+        print(e)
+    return None
+
+while True:
```
Comment (stephane-segning) [inline app/scraping.py:30]: Remove this function please @Koufan-De-King

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Good work @Koufan-De-King. But why change the app into a folder?

**Comment by Koufan-De-King** [comment]
> Good work @Koufan-De-King. But why change the app into a folder?

Good day, @stephane-segning. The app directory is to provide a good structure to our project and is useful when importing the individual functions because otherwise it returns an error.
Or sir is there an alternative way ?

File: app/scraping.py (hunk 1 above)
Comment (Koufan-De-King) [reply app/scraping.py:30]: > Remove this function please @Koufan-De-King 
Please sir @stephane-segning , what exactly do you mean? The while loop? or what?

Comment (AssahBismarkabah) [reply app/scraping.py:30]: here is the view sir

**Comment by stephane-segning** [comment]
@Koufan-De-King you should keep the `app.py` where it was already. It was not interesting to change that structure for your ticket

**Review by stephane-segning — APPROVED** [review]
Perfect!

**Pull request merged**